examples/matrix_multiplier/tests/Makefile

```diff
@@ -0,0 +1,85 @@
+TOPLEVEL_LANG ?= verilog
+SIM ?= icarus
+
+PWD=$(shell pwd)
+
+ifeq ($(OS),Msys)
+WPWD=$(shell sh -c 'pwd -W')
+else
+WPWD=$(shell pwd)
+endif
+
+# Matrix parameters
+DATA_WIDTH ?= 32
+A_ROWS ?= 10
+B_COLUMNS ?= 4
+A_COLUMNS_B_ROWS ?= 6
+
+ifeq ($(TOPLEVEL_LANG),verilog)
+    VERILOG_SOURCES = $(WPWD)/../hdl/matrix_multiplier.v
+
+    # Set module parameters
+    ifeq ($(SIM),icarus)
+        COMPILE_ARGS += -Pmatrix_multiplier.DATA_WIDTH=$(DATA_WIDTH) -Pmatrix_multiplier.A_ROWS=$(A_ROWS) -Pmatrix_multiplier.B_COLUMNS=$(B_COLUMNS) -Pmatrix_multiplier.A_COLUMNS_B_ROWS=$(A_COLUMNS_B_ROWS)
+    else ifeq ($(SIM),$(filter $(SIM),questa aldec))
+        SIM_ARGS += -gDATA_WIDTH=$(DATA_WIDTH) -gA_ROWS=$(A_ROWS) -gB_COLUMNS=$(B_COLUMNS) -gA_COLUMNS_B_ROWS=$(A_COLUMNS_B_ROWS)
+    else ifeq ($(SIM),vcs)
+        COMPILE_ARGS += -pvalue+matrix_multiplier/DATA_WIDTH=$(DATA_WIDTH) -pvalue+matrix_multiplier/A_ROWS=$(A_ROWS) -pvalue+matrix_multiplier/B_COLUMNS=$(B_COLUMNS) -pvalue+matrix_multiplier/A_COLUMNS_B_ROWS=$(A_COLUMNS_B_ROWS)
+    else ifeq ($(SIM),ius)
```

Comment:
You can change this line (and the one for VHDL) to
```
ifneq ($(filter $(SIM),ius xcelium),)
```
because the options will be the same for both simulators.
BTW, we recently arrived at the ``ifneq`` style to handle ``SIM``being empty, I guess it would be good to change the lines for questa and aldec accordingly, even if only for general consistency.